package.json

```diff
@@ -59,7 +59,7 @@
   },
   "type": "commonjs",
   "engines": {
-    "node": ">=13"
+    "node": ">=16.6"
```

Comment:
Are we OK with using 16.6? The LTS version is 16.3.2

Reply:
I believe LTS is 16.x. As per Node.js's download page: `Latest LTS Version: 16.13.2`

If I remember correctly this was required when building, for this new line added in `Message.tsx`:

https://github.com/Avansai/next-multilingual/blob/a5fbc08dc5ff20d92783b3252e2d53143602522a/src/messages/Message.tsx#L152